docs/section19/19-19.md

```diff
@@ -0,0 +1,96 @@
+**Rule Assertion:** Options are Pass/UNDETERMINED/NOT_APPLICABLE     
+**Appendix G Section:** G3.1.2.9          
+**90.1 Section Reference:** Table G3.9.1    
+
+**Data Lookup:** Table G3.9.1    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** 
+1. Applies to baseline systems 3 through 8, and 11, 12, and 13.
+
+**Function Calls:**  
+1. get_baseline_system_types()    
+2. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()   
+3. get_fan_object_electric_power()  
+4. get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM()  
+5. baseline_system_type_compare()  
+
+## Rule Logic:   
+**Applicability Check 1**   
+- Create a list of the target system types: `target_system_types = [HVAC_SYS.SYS_3, HVAC_SYS.SYS_4, HVAC_SYS.SYS_5, HVAC_SYS.SYS_6, HVAC_SYS.SYS_7, HVAC_SYS.SYS_8, HVAC_SYS.SYS_11, HVAC_SYS.SYS_12, HVAC_SYS.SYS_13]`  
+- Create dictionary of baseline system types: `baseline_system_types_dict = get_baseline_system_types(B_RMI)`  
+- Check if B-RMI is modeled with at least one air-side system that is of type associated with the target systems, if yes then carry on, if no then NOT_APPLICABLE for the project: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types):`  
+    - Create dictionary of hvac systems, zones, and terminal units: `dict_of_zones_and_terminal_units_served_by_hvac_sys = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`
+    - For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:` 
+        - Get the baseline system type: `sys_type = list(baseline_system_types_dict.keys())[list(baseline_system_types_dict.values()).index(hvac.id)])`
+        - Check if system type is of an applicable system type, if yes then carry on, if no then NOT_APPLICABLE for the hvac system:  `if any(baseline_system_type_compare(sys_type, target_sys_type, false) for target_system_type in target_system_types):`         
+            - Get the list of zones associated with the hvac system: `zone_list = dict_of_zones_and_terminal_units_served_by_hvac_sys[hvac.id]["ZONE_LIST"]`  
+            - Reset supply cfm to zero: `supply_cfm_b = 0`  
+            - Reset total_fan_power to zero: `total_fan_power = 0`   
+            - Reset fan_power_W_CFM to zero: `fan_power_W_CFM = 0`    
+            - Reset total exhaust cfm: `total_exhaust_cfm = 0`       
+            - Reset total relief cfm: `total_relief_cfm = 0`       
```

Comment:
Doesn't seem we need exhaust and releive CFM?

Reply:
CML to remove

Reply:
Implemented.

Reply:
Relief CFM still there in line 37. Not critical, but I think we agreed to remove?